src/quick-lint-js/parse.h

```diff
@@ -1263,6 +1263,14 @@ class parser {
       }
       this->skip();
 
+      if (this->peek().type == token_type::equal_greater) {
```

Comment:
This applies to `function f() => {}` too, right? If so, we should add a test case for that syntax. And we should tweak the message. (Perhaps have two different messages? One saying "functions" and one saying "methods".)

Same with `class C { f() => {} }`, except those are called methods, so the current wording works.

Reply:
> This applies to `function f() => {}` too, right?

Right



> Perhaps have two different messages? One saying "functions" and one saying "methods

Do you mean two different errors? Also how do we know if we are in a function or method?

Reply:
> Do you mean two different errors?

Yes, or a single error type with a parameterized message. I'm leaining toward two different error types.

> Also how do we know if we are in a function or method?

Dunno.